### PR TITLE
docs: update pruning docs

### PR DESCRIPTION
**Pull request by evan-forbes**
## Overview

This PR adds some descriptions to each node type's recommended pruning configurations.

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: nodes/consensus-node.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ Here are the summarized recommendations for each node type. There are more detai
 
 ##### Validator node
 
-The reccomendations here are assuming that the validator node is isolated from other responsiblities other than voting and proposing. It is optimized to store as little data as possible. This means that we're retaining the block data for the past ~3 weeks, not indexing transaction, storing the results of the execution of txs, and it's only storing the past two state snapshots. Note that if the validators are connected to a bridge node then the [Bridge Node](#bridge-node) configuration should be used.
+The reccomendations here are assuming that the validator node is isolated from other responsiblities other than voting and proposing. This means that it is not indexing transaction, storing the results of the execution of txs, and it's only storing the past two state snapshots. Note that if the validators are connected to a bridge node then the [Bridge Node](#bridge-node) configuration should be used.
```
Comment (musalbas) [inline nodes/consensus-node.md:155]: This doesn't have indexer = null etc?

**Comment by evan-forbes** [comment]
people can feel free to push to this PR to push it over the line

File: nodes/consensus-node.md [hunk 2]
```diff
@@ -152,7 +152,7 @@ Here are the summarized recommendations for each node type. There are more detai
...
 
 ```toml
```
Comment (musalbas) [inline nodes/consensus-node.md:157]: Can we add what file this is for like below?

**Comment by jcstein** [comment]
sounds g. i removed reviews bc realized i added them prematurely

**Comment by musalbas** [comment]
>This means storing all the block data by setting the min-retain-blocks = 0, but pruning past the last 10 state snapshots.

Shouldn't this say _pruning all but the last 10 state snapshots_?

**Comment by jcstein** [comment]
# [Preview of page](https://celestiaorg.github.io/docs/pr-preview/pr-1214/nodes/consensus-node)

**Comment by musalbas** [comment]
lgtm

File: nodes/consensus-node.md (hunk 2 above)
Comment (evan-forbes) [reply nodes/consensus-node.md:169]: yeah good catch added in 7a43dc82ea08b452a085d415b1f1d98b1d96b7c9

**Comment by evan-forbes** [comment]
> Shouldn't this say pruning all but the last 10 state snapshots?

good point got rid of the double negative in https://github.com/celestiaorg/docs/pull/1214/commits/fd64e3a0af118bc62e403835e04085457366d9b4

**Comment by musalbas** [comment]
lgtm

**Review by jcstein — APPROVED** [review]
:shipit:

**Pull request merged**